pcsx2-qt/Settings/GraphicsSettingsWidget.ui

```diff
@@ -1611,7 +1611,7 @@
             <item row="0" column="1">
              <widget class="QCheckBox" name="osdShowSpeed">
               <property name="text">
-               <string>Show Speed</string>
+               <string>Show Emulation Speed</string>
```

Comment:
iirc this is both the emulation speed and the game speed (scaled ofc)

Reply:
You are right. Correcting.

Reply:
Actually expanded the string, what do you think about "Show Emulation/Game Speeds"?

Reply:
`Actually expanded the string, what do you think about "Show Emulation/Game Speeds"?` It only shows percentages for both. Maybe just say `Show Speed Percentages` instead of that long text.

Reply:
Done.